runtime/doc/filetype.txt

```diff
@@ -510,63 +510,82 @@ Local mappings:
-Man {sect} {name}         Display the manpage for {name} and section {sect}.
-Man {name}({sect})        Same as above.
-Man {sect} {name}({sect}) Used during completion to show the real section of
-                          when the provided section is a prefix, e.g. 1m vs 1.
-Man {path}                Open the manpage at {path}. Prepend "./" if {path}
-                          is relative to the current directory.
-Man                       Open the manpage for the <cWORD> (man buffers)
-                          or <cword> (non-man buffers) under the cursor.
-Man!                      Display the current buffer contents as a manpage.
-
-|:Man| accepts command modifiers. For example, to use a vertical split: >
-     :vertical Man printf
+:Man {sect} {name}	Display the man page for {name} and section {sect}.
+
+:Man {name}({sect})	Same as above.
+
+:Man {name}.{sect}	Same as above.
+
+:Man {path}		Open the man page at {path}.  Prepend "./" or "../"
+			for relative paths.
+
+:Man!			Display the current buffer contents as a man page.
+
+The three first forms above provide completion for the {name} argument.  The
+'fileignorecase' option allows you to control the case sensitivity of the
+completion.
+
+Command modifiers may be used.  For example, to open in a vertical split: >
+	:vertical Man printf
+
+|:Man| also generates tags for man pages with prefixes matching {name} within
+a section.  You can use tag commands like |:tselect| to easily access these
+other man pages. |tagsrch.txt|
 
 Local mappings:
-K or CTRL-]               Jump to the manpage for the <cWORD> under the
-                          cursor. Takes a count for the section.
-CTRL-T                    Jump back to the location that the manpage was
-                          opened from.
-gO                        Show the manpage outline. |gO|
-q                         :quit if invoked as $MANPAGER, otherwise :close.
+K  or CTRL-]	Jump to the man page for the <cWORD> under the
+		cursor.  Takes a count for the section.
+
+CTRL-T		Jump back to the location that the man page was
+		opened from.  |CTRL-T|
+
+gO		Show the man page outline.  |gO|
+
+q		":quit" if invoked as $MANPAGER, otherwise ":close".
 
 Variables:
-*g:no_man_maps*             Do not create mappings in manpage buffers.
-*g:ft_man_folding_enable*   Fold manpages with foldmethod=indent foldnestmax=1.
-*b:man_default_sects*       Comma-separated, ordered list of preferred sections.
-                          For example in C one usually wants section 3 or 2: >
-                               :let b:man_default_sections = '3,2'
-*g:man_hardwrap*            Hard-wrap to $MANWIDTH or window width if $MANWIDTH is
-                          empty. Enabled by default. Set |FALSE| to enable soft
-                          wrapping.
-
-To use Nvim as a manpager: >
-     export MANPAGER='nvim +Man!'
-
-Note that when running `man` from the shell and with that `MANPAGER` in your
```

Comment:
This changes are really hard to review unfortunately, I don't know what you changed and why without manually diffing everything. Could you elaborate and justify them?

Reply:
This also doesn't document the tagfunc so it doesn't deal with the linked issue.

Reply:
I'm a bit confused with this review-mode of Github. I thought I already answered this question somewhere else?

Reply:
In short,

- I move the paragraphs so that the related information is closer together (easier to grok).
- I added vertical spacing between descriptions to make it easier to read (and also for consistency with other help pages).
- I changed the indentation from spaces to tabs for consistency with the rest of the file and other help files.